### PR TITLE
Move and rename security packages

### PR DESCRIPTION
**Pull request by arjantijms**
Rename main package in security-ee module to include .ee

Before the security and security-ee modules both had the same main package name.

Rename auth package to authentication in .ee package, and move the GlassFish LoginModule/Realms combinations to a new package.

Rename JDBCDigest which had nothing to do with JDBC.

### CODE REVIEW

File: appserver/tests/admin/tests/src/test/java/org/glassfish/main/admin/test/ClusterITest.java [hunk 1]
```diff
@@ -60,6 +61,7 @@
  * @author Tom Mueller
  */
 @TestMethodOrder(OrderAnnotation.class)
+@Disabled
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/admin/tests/src/test/java/org/glassfish/main/admin/test/ClusterITest.java]: Good that you found it, that should not have been committed.

File: appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecurityContainer.java [hunk 2]
```diff
@@ -26,16 +26,15 @@
...
+import com.sun.enterprise.security.ee.web.integration.WebSecurityManagerFactory;
 import jakarta.inject.Inject;
 import jakarta.inject.Provider;
 
 /**
  * Security container service
  *
  */
-@Service(name = "com.sun.enterprise.security.ee.SecurityContainer")
+@Service(name = "com.sun.enterprise.security.ee.ee.SecurityContainer")
```
Comment (avpinchuk) [inline appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecurityContainer.java:37]: @arjantijms, it looks like the some bad thinh - double `ee` in package name

File: appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecuritySniffer.java [hunk 3]
```diff
@@ -41,7 +41,7 @@
 @Service(name = "Security")
 public class SecuritySniffer extends GenericSniffer {
 
-    final String[] containers = { "com.sun.enterprise.security.ee.SecurityContainer" };
+    final String[] containers = { "com.sun.enterprise.security.ee.ee.SecurityContainer" };
```
Comment (avpinchuk) [inline appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecuritySniffer.java:44]: Here too

File: appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecurityContainer.java (hunk 2 above)
Comment (arjantijms) [reply appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecurityContainer.java:37]: Thanks, fixed it!

**Comment by avpinchuk** [comment]
@arjantijms, `appserver/web/web-glue` contains `com.sun.enterprise.security.web` package. Maybe cause here?

**Comment by arjantijms** [comment]
> @arjantijms, `appserver/web/web-glue` contains `com.sun.enterprise.security.web` package. Maybe cause here?

Might be, the problem seems to be the other way around. It manually exports packages `com.sun.enterprise.security.web` was globally renamed to `com.sun.enterprise.security.ee.web`, but there it should not be renamed.

File: appserver/security/core-ee/src/test/resources/com/sun/enterprise/security/ee/perms/server.policy [hunk 4]
```diff
@@ -99,7 +99,7 @@ grant {
 
 // Permissions to invoke CORBA objects in server
 grant {
-    permission com.sun.enterprise.security.CORBAObjectPermission "*", "*";
+    permission com.sun.enterprise.security.ee.CORBAObjectPermission "*", "*";
```
Comment (avpinchuk) [inline appserver/security/core-ee/src/test/resources/com/sun/enterprise/security/ee/perms/server.policy:102]: Fix, please here

File: appserver/security/core-ee/pom.xml [hunk 5]
```diff
@@ -213,7 +213,7 @@
                 <configuration>
                     <archive>
                         <manifestEntries>
-                            <probe-provider-class-names>com.sun.enterprise.security.auth.realm.RealmsProbeProvider,com.sun.enterprise.security.WebSecurityDeployerProbeProvider,com.sun.enterprise.security.EjbSecurityPolicyProbeProvider</probe-provider-class-names>
+                            <probe-provider-class-names>com.sun.enterprise.security.ee.auth.realm.RealmsProbeProvider,com.sun.enterprise.security.ee.WebSecurityDeployerProbeProvider,com.sun.enterprise.security.ee.EjbSecurityPolicyProbeProvider</probe-provider-class-names>
```
Comment (avpinchuk) [inline appserver/security/core-ee/pom.xml:216]: Please, view here. You do not touch nucleus classes

File: appserver/security/core-ee/src/test/resources/com/sun/enterprise/security/ee/perms/server.policy (hunk 4 above)
Comment (arjantijms) [reply appserver/security/core-ee/src/test/resources/com/sun/enterprise/security/ee/perms/server.policy]: Thanks!

**Review by avpinchuk — APPROVED** [review]

**Pull request merged**